src/constants/grid.ts

```diff
@@ -11,10 +13,10 @@ interface Grid {
 const grid: Grid = {
   breakpoints: {
     xs: 0,
-    l: 992,
-    m: 768,
-    s: 576,
-    xl: 1300,
+    l: breakpoints.desktop,
+    m: breakpoints.tablet,
+    s: breakpoints.phone,
+    xl: breakpoints.largeDesktop,
```

Comment:
```suggestion
    xl: breakpoints.largeDesktop,
    l: breakpoints.desktop,
    m: breakpoints.tablet,
    s: breakpoints.phone,
```
sorry OCD to keep things ordered 😛